customer-service-tests/CustomerFactoryTests.cs

```diff
@@ -51,16 +51,30 @@ public void CustomerFactorySetsActiveCustomer()
             CustomerFactory.Instance.ActiveCustomer = customer2;
 
             Assert.Equal("Matt", CustomerFactory.Instance.ActiveCustomer.FirstName);
-        }
-
-        [Fact]
+        }
+
+        [Fact]
         public void CustomerFactoryWillGetSingleCustomerFromDatabase()
         {
             CustomerFactory factory = new CustomerFactory();
             Customer customer = factory.get(1);
             Assert.Equal("Steve", customer.FirstName);
             Assert.Equal("Brownlee", customer.LastName);
             Assert.Equal(1, customer.CustomerId);
-        }
-    }
-}
+        }
+
+        [Fact]
+        public void CustomerFactoryWillGetMultipleCustomersFromDatabase()
+        {
+            CustomerFactory factory = new CustomerFactory();
+            List<Customer> customers = factory.getAll();
+            Assert.NotEmpty(customers);

```

Comment:
maybe also assert that the count is greater than 1:
`Assert.True(customers.Count() > 1)` or something like that